internal/proto/controller/storage/servers/store/v1/worker_auth.proto

```diff
@@ -0,0 +1,66 @@
+syntax = "proto3";
+
+// Package store provides protobufs for storing types in the pki package.
+package controller.storage.servers.store.v1;
+
+import "controller/storage/timestamp/v1/timestamp.proto";
+
+option go_package = "github.com/hashicorp/boundary/internal/servers/store;store";
+
+// WorkerAuth contains all fields related to an authorized Worker resource
+message WorkerAuth {
+  // The key id for this worker auth record, generated from the signing pub key
+  // @inject_tag: `gorm:"primary_key"`
+  string worker_key_identifier = 10;
+
+  // The worker id this worker authentication record is for
+  string worker_id = 20;
+
+  // The worker's public signing key
+  bytes worker_signing_pub_key = 30;
+
+  // The worker's public encryption key
+  bytes worker_encryption_pub_key = 40;
+
+  // The private key associated with this certificate
+  // This is a ciphertext field
+  bytes controller_encryption_priv_key = 50;
+
+  // controller_encryption_priv_key_hmac is a sha256-hmac of the unencrypted controller_encryption_priv_key
+  // It is recalculated everytime controller_encryption_priv_key is updated
+  bytes controller_encryption_priv_key_hmac = 60;
+
+  // The id of the kms database key used for encrypting this entry.
+  string key_id = 70;
+
+  // Nonce used by a worker in authenticating
+  bytes nonce = 80;
+}
+
+// WorkerCertBundle contains all fields related to a WorkerCertBundle resource
+message WorkerCertBundle {
+  // The serial number of the issuing root certificate
+  uint64 issuing_certificate = 10;
+
+  // The WorkerAuth worker_key_identifier this cert bundle record is for
+  string worker_key_identifier = 20;
+
+  // Certificate is the PEM encoded certificate.
```

Comment:
I assume this is actually just a marshaled cert bundle from `nodeenrollment`? In which case it's not PEM encoded, it's a marshaled protobuf.